primers_test.go

```diff
@@ -80,3 +81,51 @@ func TestMeltingTemp(t *testing.T) {
 		t.Errorf("MeltingTemp has changed on test. Got %f instead of %f", calcTM, expectedTM)
 	}
 }
+
+func ExampleDeBruijn() {
+	a := DeBruijn(4)
+
+	fmt.Println(a)
+	// Output: AAAATAAAGAAACAATTAATGAATCAAGTAAGGAAGCAACTAACGAACCATATAGATACATTTATTGATTCATGTATGGATGCATCTATCGATCCAGAGACAGTTAGTGAGTCAGGTAGGGAGGCAGCTAGCGAGCCACACTTACTGACTCACGTACGGACGCACCTACCGACCCTTTTGTTTCTTGGTTGCTTCGTTCCTGTGTCTGGGTGGCTGCGTGCCTCTCGGTCGCTCCGTCCCGGGGCGGCCGCGCCCCAAA
+}
+
+func ExampleUniqueSequence() {
+	uniqueSequences := make(chan string)
+	go UniqueSequence(uniqueSequences, 20, 4, []string{"CTCTCGGTCGCTCC"}, []func(string) bool{})
+	var output string
+	for uniqueSeq := range uniqueSequences {
+		output = uniqueSeq
+	}
+
+	// Only we only want 1 output to check against
+	fmt.Println(output)
```

Comment:
```suggestion
	fmt.Println(<-uniqueSequences)
```
If we just check the first item, we can avoid the for loop here.

Reply:
Have you tried running that exact code? I get a panic when I do